.github/workflows/bazel-cache-push.yml

```diff
@@ -56,23 +56,25 @@ jobs:
       - name: Bring up the Magma VM
         run: |
           cd lte/gateway
+          export MAGMA_DEV_CPUS=3
+          export MAGMA_DEV_MEMORY_MB=9216
```

Comment:
I assume this is intentionally 8GB+1GB to allow the overhead needed for the JVM or such?
Just want to make sure...

Reply:
Right - see https://github.com/magma/magma/pull/12524 and the related description/analysis https://github.com/magma/magma/pull/12501